Makefile

```diff
@@ -2,30 +2,27 @@ SHA = $(shell git rev-parse --short HEAD)
 GO_PKGS = $$(go list ./... | grep -v vendor)
 REPOSITORY = 095427547185.dkr.ecr.us-west-2.amazonaws.com/skuid/skuid
 TAG = latest
-VOL_PATH=/go/src/github.com/skuid/tides
-GO_VERSION=1.15
+VOL_PATH=/go/src/github.com/skuid/skuid-cli
+GO_VERSION=1.18
 IMAGE ?= golang
 
 ARCH=amd64
 OS=darwin
 
 VERSION=`cat .version`
-LDFLAGS=-ldflags="-w -X github.com/skuid/tides/version.Name=$(VERSION)"
+LDFLAGS=-ldflags="-w -X github.com/skuid/skuid-cli/version.Name=$(VERSION)"
 
 .PHONY: setup fmt vendored
 
 # fails if coverage < 85%
 # see MainTest for details
 # (TODO: add coverfail=true)
-ci:
-	go test -v -cover ./... -args 
+cover:
```

Comment:
![image](https://user-images.githubusercontent.com/30167939/202042069-4f65dd66-6599-428b-a37c-26304700badc.png)
cover is generating a warning